root/statistics/types.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+export type StatsT = {|
+  +data: {[string]: number},
+  +date_collected: string,
```

Comment:
You already pass a `dateCollected` prop, so I'm not sure why this needs to be in this object. Or rather, why `StatsT` isn't just `{[string]: number}`, so that instead of having to do `stats.data['foo']` everywhere, you can just do `stats['foo']`.

Reply:
And if it's just `{[string]: number}` I would delete this type and this file, since you can inline `{[string]: number}` in the one place it's used.